msg/sensor_gps.msg

```diff
@@ -1,19 +1,36 @@
-# EKF blended position in WGS84 coordinates.
+# GPS position in WGS84 coordinates.
+# the field 'timestamp' is for the position & velocity (microseconds)
 uint64 timestamp		# time since system start (microseconds)
+
 int32 lat			# Latitude in 1E-7 degrees
 int32 lon			# Longitude in 1E-7 degrees
 int32 alt			# Altitude in 1E-3 meters above MSL, (millimetres)
 int32 alt_ellipsoid 		# Altitude in 1E-3 meters bove Ellipsoid, (millimetres)
+
 float32 s_variance_m_s		# GPS speed accuracy estimate, (metres/sec)
+float32 c_variance_rad		# GPS course accuracy estimate, (radians)
```

Comment:
as @mhkabir said, it's the right time to fix #6275

Reply:
The discussion doesn't seem finished, are you suggesting the simple naming change?
```
float32 s_accuracy_m_s		# GPS speed accuracy estimate, (metres/sec)
float32 c_accuracy_rad		# GPS course accuracy estimate, (radians)
```

Reply:
I'd be happy to implement additional fixes, however I don't fully understand what changes you guys are requesting as they aren't part of the direct subject matter of this PR. I suggested @mhkabir to push a commit up to fix them as he clearly knows exactly what he'd like done, as I see he has a [PR](https://github.com/PX4/Firmware/compare/master...mhkabir:gps_refactor?expand=1) (which is quite large) that is referenced in the issue #6275

Reply:
`s_accuracy_m_s` and `c_accuracy_rad` are good names IMO

Reply:
This has to be changed in coordination with the GpsDrivers submodule. https://github.com/PX4/GpsDrivers

Let's do it separately.